src/util.ts

```diff
@@ -48,10 +49,26 @@ export function getMinikubePath(): string {
   }
 }
 
+export function getMinikubeHome(): string {
+  const minikubeHome = minikubeConfiguration.get<string>('home');
+  // Check env if configuration is not applied in UI
+  if (!minikubeHome) {
+    const env = process.env;
+    return env.MINIKUBE_HOME;
+  } else {
+    return minikubeHome;
+  }
+}
+
```

Comment:
the function in case of missing env var will return undefined so the signature seems invalid (should be `string | undefined`)